.github/workflows/CI-CD.yml

```diff
@@ -0,0 +1,46 @@
+name:  CI/CD
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    services:
+      # Label used to access the service container
+      redis:
+        # Docker Hub image
+        image: redis
+        # Set health checks to wait until redis has started
+        options: >-
+          --health-cmd "redis-cli ping"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+        ports:
+          - 6379:6379
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.14.2
+      - name: Install dependencies
+        run: npm ci
+      - name: Run tests
+        run: npm run test
+
+  publish:
+    if: github.ref == 'refs/heads/main'
+    needs: test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      # Setups .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.14.2'
+          registry-url: 'https://registry.npmjs.org'
+      - run: npm ci
+      - run: npm publish
```

Comment:
I  think you need to add 
- run: npm run build

before publish command

Reply:
Thanks I managed to fix the package json and the package no longer breaks on npm RunKit web browser command line!!!!

Reply:
Exported to impact graph locally and it didn't break.